content/billing-pricing/pricing.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Pricing
```

Comment:
For the record, currently there are no links from Billing page to this page, the Billing page refers to the pricing page on our landing for pricing related info.